src/extension.ts

```diff
@@ -78,6 +78,16 @@ let decorationType: TextEditorDecorationType = window.createTextEditorDecoration
   }
 );
 
+const jdk8 = "1.8.232";
+const jdk8Action = "JDK8";
+
+const jdk11 = "1.11";
```

Comment:
are you sure about this? We recently fixed a bug due to Java 11 being on version “11” rather than “1.11”

Reply:
It's coming from https://github.com/shyiko/jabba/blob/master/index.json

Renamed it to `jdk11Jabba` to pronounce it better.